maintenance/scripts/refresh_common.sh

```diff
@@ -7,6 +7,6 @@ cd /var/www/html/frontend
 npm install
 
 cd /var/www/html/frontend/common
-(cd .. && npm run h2-build --workspace common)
+(cd ..;node node_modules/@hydrogen-design-system/hydrogen.css/bin/build.js)
```

Comment:
Why is this line not included in the refresh script for admin/iap/ts?

Reply:
Also, why isn't this just set as h2-build in the package.json file like before?

Reply:
Probably the other h2-* scripts should be removed from package.json.

Reply:
1) Because Hydrogen is run during `npm run dev` now, so they don't need it
2) Because of weird shell issues @patcon and I couldn't figure out. No shell extension can run NPM in our repo for some reason, despite me being able to get it to work in a webpack/react boilerplate.

Reply:
using `node` directly was just a way to side-step some PATH related complexity that npm was introducing, and move past it. It's not a better way, per se :)

i don't recall if it was just on your workstation or CI josh, but if so, it might be resolved when you deleted your lingering `frontend/common/node_modules`

🤔 Would it be worth adding a very robust `npm run clean` command to do a total reset (akin to fresh clone), so we have a standard way of deleting/archiving gitignored files and non-version-controlled things. Quirks of gitignored stuff is probably our main source of "works on my machine" situations.